packages/css-data/src/property-parsers/transition-property-extractor.ts

```diff
@@ -0,0 +1,55 @@
+import type {
+  KeywordValue,
+  TupleValue,
+  UnitValue,
+} from "@webstudio-is/css-engine";
+
+const isTimingFunction = (timing: string) => {
+  const regex =
+    /^(ease(-in-out|-in|-out)?|linear|cubic-bezier\((-?\d+(\.\d+)?, ?){3}-?\d+(\.\d+)?\)|steps\(\d+(-?(start|end))?\))$/gm;
```

Comment:
this is scary, what are your thoughts about using CSS parser vs this?

Reply:
We are using css parser for initially parsing the value and finding individual values. But it just returns `Identfier` which can be anything there since transition is a complex value made up of two keyword values. Used this regex to find out the property from the kyewords.

Reply:
there are css value parsers, I have a feeling we need one, this thing is gonna bite us badly

Reply:
also there needs to be tests for all of the things this regex covers if we are not using a library, but  I still think we should use a proper library to parse this @istarkov wdyt?

Reply:
I would use css-tree and take function name to parse
<img width="1522" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/062533c4-c98b-421b-8740-0b8a4d216353">

Reply:
Yup, but it only works for `cubic-beizer` `setps`. For `ease`, `ease-in` etc it again shows as Identifiers. Which need to be verified again as both transition-property and timing functions are regarded as `identifiers` in those case. 

<img width="1799" alt="Screenshot 2023-11-04 at 21 42 07" src="https://github.com/webstudio-is/webstudio/assets/11075561/356b2633-67ce-4182-b12c-4f275a63f32e">


So, still need to maintain a record of identifers and function names which are inside regex.  
And it will be two step process, regex-parse + name check from a list.
So i don't see much upside by moving from regex check to parsing using a tree and still comparing with a array. Anything specific with regex or it more code clarity ? 🤔

Reply:
Resolving here, moving the discussion to the transition-timing PR. Since these are related and it's a function swap anyways 👍